example/backends/spidsaml2_validator.py

```diff
@@ -280,6 +280,13 @@ def validate_assertion_authn_statement(self):
                         f'{_ERROR_TROUBLESHOOT}'
                     )
 
+                # 94, 95, 96
+                if authns.authn_context.authn_context_class_ref.text != self.authn_context_class_ref:
```

Comment:
update here:

https://github.com/italia/spid-django/commit/5a44b91a1d6b1fe9147b88b2aab7d74a47c0de4a#diff-767d8eb2d2b8ee3481657fe7595b6356a60eef1efd6a85c6f3411ec1b3780139

TODO also in SATOSA, exactly here :)

Reply:
Added :+1: